frontend/src/views/v-authorship.vue

```diff
@@ -119,21 +127,31 @@
                 font-awesome-icon.button(icon="user-edit")
                 span.tooltip-text Click to view the blame view of file
           span.fileTypeLabel(
-            v-if='!file.isBinary',
+            v-if='!file.isBinary && !file.isIgnored',
             v-bind:style="{\
               'background-color': fileTypeColors[file.fileType],\
               'color': getFontColor(fileTypeColors[file.fileType])\
               }"
           ) {{ file.fileType }}&nbsp;{{ file.lineCount }}
             |&nbsp;({{ file.lineCount - file.blankLineCount }})
           span.fileTypeLabel.binary(v-if='file.isBinary') binary&nbsp;
-        pre.hljs.file-content(v-if="file.wasCodeLoaded && !file.isBinary", v-show="file.active")
+          span.fileTypeLabel(
+            v-if='file.isIgnored',
+            v-bind:style="{\
+              'background-color': '#000000',\
+              'color': '#ffffff'\
+              }"
```

Comment:
v-bind style is for dynamic styles, this isn't dynamic so it's better to use a class and place the css in the css section.

Reply:
@gerhean does the fix look better?